backend/handler/filesystem/roms_handler.py

```diff
@@ -123,34 +275,15 @@ def get_roms(self, platform: Platform):
         ]
 
         return [
-            dict(
-                rom,
+            FSRom(
+                multi=rom["multi"],
+                file_name=rom["file_name"],
                 files=self.get_rom_files(rom["file_name"], roms_file_path),
```

Comment:
Is this call to `get_rom_files` making us read all the rom files and calculate the hashes, every time a scan is triggered?

If that's the case, we need to find a way to only calculate hashes when it's the first time the file has been scanned, or its `mtime` has been updated.

Reply:
alright i cleaned up a lot of stuff, most notably it'll only calculate hashes when adding new roms or on complete re-scans.